features/guidance/collapse.feature

```diff
@@ -755,12 +755,13 @@ Feature: Collapse
     Scenario: Don't collapse going straight if actual turn
         Given the node map
             """
-              c e
-                d   f
-
+                e
+             c  |
+              \ d - - - f
+               \|
                 b
-
-
+                |
+                |
                 a
             """
```

Comment:
this tests (due to the move of c) is also changed for turn-angles. The widening of `straight` in `getTurnDirection` would have altered the purpose of the test otherwise, interacting with https://github.com/Project-OSRM/osrm-backend/blob/12ded539aaba3f730638ab008db1a3ad3c0c345a/src/extractor/guidance/turn_handler.cpp#L192-L195